### ISSUE TITLE
final widget not being rendered

### ISSUE THREAD

**Issue by jaycee14** [janfreyberg/superintendent#54]
Hello,

I'm looking to extend superintendent and have managed all my custom changes but at the end of a labeling cycle the status bar turns green but nothing else happens and a press on any button raises a stopIteration error.

From the code (below) it looks like a widget should appear overlaying the existing buttons so no more user interaction can occur - but it doesn't and there are no error messages to chase down either.

I have python 3.7.7 and ipywidgets 7.5.1 installed

```
    def _render_finished(self):

        self.progressbar.bar_style = "success"   # <- this happens

        finish_celebration = widgets.Box(
            (widgets.HTML(value="<h1>Finished labelling!"),),
            layout=widgets.Layout(
                justify_content="center",
                padding="2.5% 0",
                display="flex",
                width="100%",
            ),
        )
        self.layout.children = [self.progressbar, finish_celebration]  # <- this never appears
```

**Comment by janfreyberg** [comment]
Hi there,

Thanks for this bug report! You're right, that particular line is wrong - I changed the widgets from having a "layout" attribute to just having the "children" attribute set separately. I'll fix this now as it's a very small change.

Thanks

**Comment by janfreyberg** [comment]
This is now released in version 0.5.2.

**Comment by jaycee14** [comment]
Great - thanks!
